packages/hardhat-ethers/src/helpers.ts

```diff
@@ -1,4 +1,4 @@
-import { ethers } from "ethers";
+import type { ethers } from "ethers";
```

Comment:
Pretty much equivalent in this case, but I think this should be `import type ethers from "ethers";`

Reply:
Good point! It should be the entire module.

I came up with this new dynamic imports pattern doing this PR, where you do:

```ts
import type mod from "mod";

function f() {
  const { thing } = require("mod") as typeof mod;
  thing();
}
```

It's way less cumbersome than what I used to do, but it may not be as straight forward if the modules has an `exports default`.